docker/api/Dockerfile

```diff
@@ -4,29 +4,33 @@ FROM debian:buster
 USER root
 
 # Install system packages
-RUN apt-get update && \
-    apt-get -y dist-upgrade && \
-    apt-get install -y \
+RUN apt-get update \
+    && apt-get -y upgrade \
```

Comment:
I think both `dist-upgrade` and `upgrade` are redundant. If you need the latest Debian updates, you only need to pull a new version of `debian:buster` from the Dockerhub.

In fact, if you always issue docker build commands with the `--pull` option, Docker will check if a newer version of the base image is available on the Dockerhub and pull and rebuild from scratch if there is one.